Tasks/HelmInstallerV0/src/helmtoolinstaller.ts

```diff
@@ -22,6 +24,7 @@ async function configureKubectl() {
 
 async function configureHelm() {
     var version = await helminstaller.getHelmVersion();
```

Comment:
rename to helmversion and declare it has global variable instead of new variable.